test/graph.jl

```diff
@@ -0,0 +1,69 @@
+
+
+# the test graph being loaded is labeled as
```

Comment:
For long comments, I prefer the multi-line comment style—that way, autodoc tools can pick them up.

Reply:
will do